tools/c7n_mailer/c7n_mailer/sns_delivery.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2016 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from boto3 import Session
+from utils import get_message_subject, get_resource_tag_targets, get_rendered_jinja
+
+
+class SnsDelivery(object):
```

Comment:
break off Sns related functions into it's own class, the other file was too big for awhile